src/content/docs/en/guides/styling.mdx

```diff
@@ -514,6 +514,30 @@ You can apply global styles to your Markdown content by adding [`<style>` tags](
 
 If you are using Tailwind, the [typography plugin](https://tailwindcss.com/docs/typography-plugin) can be useful for styling Markdown.
 
+## Production
+
+### Bundle control
+
+When Astro builds your site for production deployment it combines your CSS into chunks. Each page on your site is its own chunk, and CSS that is shared between multiple pages are split off into their own chunks.
+
+In each of your HTML files there will be `<link rel="stylesheet">` tags added for each of the chunks that the pages needs.
+
+In a site with a large number of pages and many different shared pieces of styles this can some times lead to many requests for styles. Astro has a way to minimize requests through the `inlineStylesheets` option. Using this option Astro will put the (minimized) stylesheet into a `<style>` tag rather than request it externally.
```

Comment:
```suggestion
Sites with a large number of pages and many different shared styles can lead to many stylesheet requests and affect site performance. You can configure the `inlineStylesheets` option to reduce the number of these requests by putting (minimized) stylesheets into a `<style>` tag rather than request them externally.
```
I think "shared styles" or even "shared chunks" is OK here. 

Also, I think we can be tighter about saying it can lead to many requests to get to the reason that's actually a problem. (I've generically thrown in "site performance" there, but you could describe exactly what issue is faced such as a slower build time or slower page load time.)